src/Symfony/Component/Mime/MimeTypes.php

```diff
@@ -1297,10 +1329,13 @@ public function guessMimeType(string $path): ?string
         'font/collection' => ['ttc'],
         'font/otf' => ['otf'],
         'font/ttf' => ['ttf'],
-        'font/woff' => ['woff', 'woff2'],
```

Comment:
I'm not sure we should remove existing extensions

Reply:
it's not been removed: it's been split in two distinct mime type, one for each version.

Reply:
I had a second look and I think we should keep the work from the upstream source. They made the split and there should be a reason for it.